src/content_delegate.js

```diff
@@ -268,9 +256,6 @@ ContentDelegate.prototype.handleDocketDisplayPage = async function () {
   );
   if (radioDateInputs.length > 1) { return; };
 
-  // if you've already uploaded the page, return
-  if (history.state && history.state.uploaded) { return; }
```

Comment:
I removed this validation from this line because It would end this function early if the user uploaded the page before, The early exit fired by this validation was an issue because the logic to insert the 'RECAP actions' button is a few lines below this statement.

This validation is still in this method but it's closer to the **uploadDocket** request (I inserted this statement in line 296). 

The new order in the validations will allow the extension to always insert the RECAP button and upload a page only once ( line 296 will terminate the execution if the user already uploaded the page).